grove/alpha/fermion_transforms/tests/test_bravyi_kitaev.py

```diff
@@ -16,8 +16,8 @@ def test_hardcoded_transform():
     x = bkt.kill(9)
     y = bkt.create(9)
 
-    assert str(x) == '0.5j*Z7*Y9*X11*X15 + 0.5*Z7*Z8*X9*X11*X15'
-    assert str(y) == '-0.5j*Z7*Y9*X11*X15 + 0.5*Z7*Z8*X9*X11*X15'
+    assert str(x) == '0.5j*Z7*Y9*X11*X15 + (0.5+0j)*Z7*Z8*X9*X11*X15'
```

Comment:
This should not be part of the PR for Grover's

Reply:
@marcusps Sure, it was breaking tests. I'll make a separate PR so that tests pass.